### ISSUE TITLE
Plain text error output from xcrun is not handled

### ISSUE THREAD

**Issue by jasonrm** [electron/notarize#177]
The JSON parsing of the xcrun output added in #173 doesn't handle an unaccepted program license agreement with Apple.

with v2.2.1
```
  ⨯ Unexpected token E in JSON at position 0  failedTask=build stackTrace=SyntaxError: Unexpected token E in JSON at position 0
    at JSON.parse (<anonymous>)
    at /Users/runner/work/repo/node_modules/@electron/notarize/src/notarytool.ts:79:25
    at Generator.next (<anonymous>)
    at fulfilled (/Users/runner/work/repo/node_modules/@electron/notarize/lib/notarytool.js:28:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

with v2.1.0
```
  ⨯ Failed to notarize via notarytool
Error: HTTP status code: 403. A required agreement is missing or has expired. This request requires an in-effect agreement that has not been signed or has expired. Ensure your team has signed the necessary legal agreements and that they are not expired.
  failedTask=build stackTrace=Error: Failed to notarize via notarytool
```

**Comment by ericvanbuhler** [comment]
I'm experiencing the same issue

**Comment by Makio64** [comment]
same issue here

**Comment by yingchen-liu** [comment]
Same here

**Comment by rotu** [comment]
This should be fixed by #191
